src/App.css

```diff
@@ -1,26 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+body {
+  background-color: #fafafa;
+}
+
+/*  Navbar */
 .navBar {
-  height: 4rem;
   display: flex;
-  align-items: center;
-  padding: 0 2rem;
+  flex-direction: row;
   justify-content: space-between;
+  position: relative;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 100%;
+}
+
+.navTitle {
```

Comment:
- As suggested by the previous reviewer,

Review Comment:

Great progress on your project! 👏 I noticed that the class you implemented in your CSS file doesn't seem to be assigned to any element in the HTML file. However, I assume that it corresponds to the title element. To achieve consistency with the Zeplin design, please follow these steps:

1. Assignment: Assign the class you created in the CSS file to the appropriate element in the HTML file. If it corresponds to the title element, make sure to add the class attribute with the class name to the title element.

2. Styling: Once the class is assigned, please ensure that the styles defined in the CSS file for that class match the styles specified in the original design. This includes font family, font size, color, and any other relevant properties.

By correctly assigning the class to the appropriate element and applying the designated styles, you will achieve the desired appearance consistent with the Zeplin design. If you have any questions or need further assistance, please feel free to ask. Keep up the excellent work! 💪